frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -152,106 +201,131 @@ const NetworkAttachmentDefinitionFormBase = (props) => {
 
   return (
     <div className="co-m-pane__body co-m-pane__form">
-      <h1 className="co-m-pane__heading co-m-pane__heading--baseline">
-        <div className="co-m-pane__name">Create Network Attachment Definition</div>
-        <div className="co-m-pane__heading-link">
+      {noNetworkTypesAvailable ? (
+        <Alert variant="info" className="co-alert" isInline title="Form not available">
+          <span>
```

Comment:
I don't think you need to wrap this in a span

Reply:
You are correct. Fixed!